doc/doxygen/src/Examples.dox

```diff
@@ -64,7 +64,12 @@
  * that some example @ref asset_management_system has a plugin,
  * installed on @ref plugin_path_var).
  *
- * We will be providing an example manager implementation soon!
+ * Check out <a href="https://github.com/OpenAssetIO/Template-OpenAssetIO-Manager-Python"
```

Comment:
Not sure if this is the best way to reference these links. Also, they are not bold right now so they are hard to see.

Reply:
I think that's fine. These are both linked on the home page too ("An Introduction").

Reply:
Fixed the styling so they are bold 👍